src/main/java/de/tum/in/www1/artemis/service/dto/UserPublicInfoDTO.java

```diff
@@ -25,6 +25,8 @@ public class UserPublicInfoDTO {
 
     private String lastName;
 
+    private String email;
```

Comment:
Are you sure this is a good idea? If I read this correctly, this will expose the emails of all users of the course to a widely accessible endpoint. Maybe you should apply an additional filtering that only sets this field for instructs, but not for tutors and students.